src/main/resources/openapi/schemas/reportTitle.json

```diff
@@ -18,6 +18,10 @@
       "type": "string",
       "description": "KB title identifier"
     },
+    "ignoreTitle": {
+      "type": "boolean",
+      "description": "whether the title should simply be ignored [default: false]"
+    },
     "kbManualMatch": {
```

Comment:
I don't have a problem with that. But the code will not honor it yet. It's not persisted. So I'd have to work on this branch to finish it.

Reply:
Sure, I realised this was not a complete solution :-)

It's more of a sketch.

So, yes, it will need persisting (which I kind of hoped the OpenAPI-based toolkit would take care of based on its being in the JSON Schema but I guess not) — but more than that, you will need to actually honour this bit in ignoring the title when you come across it. (Or perhaps that already happens when `kbTitleId` is null?)

Reply:
if kbTitleId kbTItleName are omitted, that's an entry that was not matched.

Reply:
There is also kbManualMatch.. This, initially will be set to false. When the UI fixes a title (to include kbTItleId, kbTitlteName).. Then the kbManualMatch is true. Hence there are 3 cases:

1. system match (kbTitleId, kbTitleName present) and kbManualMatch false.
2. non-fixed match (kbTitleId, kbTitleName omitted) and kbManualMatch false.
3. fixed match. (kbTitleId, kbTitleName present) and kbManualMatch true.

Reply:
OK, so I think case 4 is what the UI mockup calls "ignored": kbTitleId and kbTitleName omitted, and kbManualMatch true — that is, someone has manually decided that a non-match is correct.

If you agree, then we can close this PR without merging, as I already have the ability to set the state I need.

Reply:
ok. Thx.